fuzz/lnwire/fundinglocked/funding_locked.go

```diff
@@ -0,0 +1,21 @@
+// +build gofuzz
+
+package fundinglocked
```

Comment:
What's the rationale for each of them having a new package rather than just a series of files in a blanket package?

Reply:
yeah so at the time of writing the tests you couldn't have multiple Fuzz funcs in the same package, but that's changed now so can change it to this - would make the directory look cleaner too..